sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/AbstractReactiveDatabaseIT.java

```diff
@@ -0,0 +1,175 @@
+package io.quarkus.ts.reactive;
+
+import static io.restassured.RestAssured.given;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+public abstract class AbstractReactiveDatabaseIT {
+    @Test
+    public void getAll() {
+        getApp().given()
+                .get("/library/books")
+                .then()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    @Test
+    public void testUniEndpoint() {
+        Response response = given()
+                .when().get("/library/books/1");
+        String title = response.body().asString();
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("Slovník", title);
+    }
+
+    @Test
+    public void testFind() {
+        String title = given()
```

Comment:
```
given()
                .when().get("/library/books/2")
                .then()
                .statusCode(HttpStatus.SC_OK)
                .extract().body().asString();
```

This code is repeated coulbe refactor and reused in a private method

Reply:
It will, but I don't see any gains in it.
On the over side, if we will make our tests as simple as possible and with plain control flow, they will be easier to understand and to find a source of failure when something will go wrong.